package.json

```diff
@@ -5,17 +5,17 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./src/bin/www",
-    "devstart": "set DEBUG=dev && babel-watch ./src/bin/www",
+    "devstart": "set DEBUG=dev && babel-watch ./src/bin/dev",
     "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
-    "test": "set NODE_ENV=test&& npm run reset:db && mocha -r @babel/register --recursive src/tests/ --exit --timeout 10000",
+    "test": " set NODE_ENV=test&& npm run reset-db&& mocha -r @babel/register --recursive src/tests/ --exit",
```

Comment:
why did you remove the '--timeout 10000', due to the default timeout being too small, some of our tests would fail.

Reply:
Any reason why you removed the timeout?

Reply:
When we finally have several tests setup and running, it might timeout and cause tests to fail, adding this timeout limit doesn't necessarily mean that the tests must stay this duration before completing, please add this back.